README.md

```diff
@@ -61,6 +61,31 @@ pub struct Rgb {
 }
 ```
 
+### Customizing single fields
+
+This can be particular handy if your structure uses a type that does not implement `Arbitrary` or you want to have more customization for particular fields.
+
+```rust
+#[derive(Arbitrary)]
+pub struct Rgb {
+    // set `r` to Default::default()
+    #[arbitrary(default)]
+    pub r: u8,
+
+    // set `g` to 255
+    #[arbitrary(value = "255")]
```

Comment:
I think we should be able to allow the syntax

```rust
#[arbitrary(value = 255)]
```

here. That is, just pass through any tokens on the RHS, so users could even do function calls and stuff like `value = 1 + foo()` or whatever. This should also make the `parse_assigned_value` function unnecessary.

Reply:
+1

Reply:
My inspiration was taken from [serde](https://serde.rs/field-attrs.html)
```rust
#[serde(deserialize_with = "path")] 
```
I think the reasoning for this is if the values are double quoted it's much easier to parse second and further optional attributes if necessary. 
Anyway, I will change it.

Reply:
Addressed in https://github.com/rust-fuzz/arbitrary/pull/129/commits/f73d5f2333f4004ccdd5250c198044e5102d61f9